src/visualizers/VisualizerInterface.ts

```diff
@@ -65,7 +70,20 @@ export interface VisualizerInterface<PD extends GenericParamDescription>
      * @param element HTMLElement The DOM node the visualizer was inhabit()ing
      */
     depart(element: HTMLElement): void
-
+    /**
+     * This is called when the size of the visualizer should change, either
+     * because the window is resized, or the docking configuration has changed.
+     * Visualizer writers should take care to resize their canvas and to make
+     * sure that any html elements aren't wider than the requested width.
+     * The new size is the full available size cut down to fit in the requested
+     * aspect ratio.
+     *
```

Comment:
I'm not sure where in the code to point this issue, but it has to do with resizing.  To cause it, resize the browser window on a fresh scope to something mobile-like (or use mobile view in the developer tools) in Chromium, and you get the first attached screenshot.  Then reloading while in that size configuration will produce the second attached screenshot.  Neither of them is good behaviour.  Not sure if we should fix this now or log it as a bug for later, since it's a pretty corner case for usage, unless this PR is meant to be working fully with mobile already.  Note that I think the footer is going away later in the PR chain, so this may resolve itself.

![Screenshot from 2024-06-28 17-12-02](https://github.com/numberscope/frontscope/assets/15117097/1fa0b6f2-74d3-4b6a-b9f1-c75a7be678ed)
![Screenshot from 2024-06-28 17-12-20](https://github.com/numberscope/frontscope/assets/15117097/332ed331-e87a-423b-a7d2-c005b3334f8b)